portal/models/auth.py

```diff
@@ -27,6 +28,8 @@ class AuthProvider(db.Model):
                         nullable=False)
     user = db.relationship('User')
 
+    db.UniqueConstraint('provider', 'provider_id', name='uix_1')
```

Comment:
can we give that unique constraint a more legible name, say `auth_providers_by_provider`?  also, I think your migration script may need to include adding this constraint.  compare the test db with the one migrated.

Reply:
1) Gave it the more legible name: `auth_providers_by_provider`
2) I was surprised this didn't happen automatically. I looked at another table and realized I should have set __table_args__. After I did this the migration set this automatically. Thanks for catching this.